xcell/mappers/mapper_dummy.py

```diff
@@ -168,6 +171,13 @@ def get_nl_coupled(self):
                 self.nl_coupled[-1] += self.noise_level
         return self.nl_coupled
 
+    def get_cl_coupled(self):
```

Comment:
Is it possible to add a test for this method alone?

Reply:
yep, done